src/Avalonia.Controls.ColorPicker/ColorSpectrum/ColorSpectrum.cs

```diff
@@ -95,7 +97,7 @@ public partial class ColorSpectrum : TemplatedControl
         /// <summary>
         /// Initializes a new instance of the <see cref="ColorSpectrum"/> class.
         /// </summary>
-        public ColorSpectrum()
+        public ColorSpectrum() : base()
```

Comment:
Nit: I don't think this makes any difference, the base constructor is automatically called.

Reply:
Yea, true, just done for consistency and habit